WordPress/gutenberg
# CustomSelectControl: Make chevron consistent with SelectControl

Fixes #39400

## What?

Makes the `chevronDown` icon of the CustomSelectControl component match the size and spacings of the one in SelectControl.

## Why?

For visual consistency between SelectControl and CustomSelectControl.

## How?

Make `Icon` size the same as SelectControl, and adjust the righthand spacing.

## Testing Instructions

1. `npm run storybook:dev`
2. Go to the default story for `SelectControl` to see a temporary comparison of SelectControl, CustomSelectControl, and the new 36px height CustomSelectControl.

Any optical adjustments we'd like to make?

## Screenshots or screencast 
| Before | After |
|--------|-------|
|<img width="166" alt="SelectControl, CustomSelectControl, and the new 36px height CustomSelectControl compared vertically. The chevron icon is inconsistent" src="https://user-images.githubusercontent.com/555336/161772498-a2140963-a3fd-4b50-bb0f-46c08fad4a87.png">|<img width="169" alt="The chevron is adjusted to be consistent" src="https://user-images.githubusercontent.com/555336/161771743-165d2794-94cb-47e6-b9d8-126356e30214.png">|

⭐ Reminder: The 36px height one is the most important variant moving forward. The 30px variants will be deprecated.
